File: GestaoHYS/GestaoHYS.Core/Models/Usuario.cs

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace GestaoHYS.Core.Models
{
    [Table("Usuario")]
    public class Usuario
    {
        [Column("Id")]
        [Key]
        [DatabaseGenerated
        (DatabaseGeneratedOption.Identity)]
        [Required]
        public long Id { get; set; }

        [Column("Nome")]
        public string Nome { get; set; }

        [Column("Email")]
        public string Email { get; set; }

        [Column("Senha")]
        public string Senha { get; set; }

        [Column("perfilId")]
        public int PerfilId { get; set; }
        public virtual PerfilUsuario Perfil { get; set; }

    }
}

File: GestaoHYS/GestaoHYS.API/ClientApp/src/app/_models/Usuario.ts
import { PerfilUsuario } from "./PerfilUsuario";

export class Usuario{
    id: number;
    nome: string;
    email: string;
    ativo: boolean;
    perfilId: number;
    perfil: PerfilUsuario;
    senha: string;
    senhaConfirm: string;

    constructor(usuario){
        this.id = usuario.id;
        this.nome = usuario.nome;
        this.email = usuario.email;
        this.ativo = usuario.ativo;
        this.perfilId = usuario.perfilId;
        this.senha = usuario.senha;
        this.senhaConfirm = usuario.senhaConfirm;
    }


}